### PR TITLE
[vendor] Update goio to v1.3.3

### PR DESCRIPTION
**Pull request by aravindhp**
Updated goio entry in go.mod and ran "go mod tidy && go mod vendor"

### CODE REVIEW

**Review by sebsoto — APPROVED** [review]

**Comment by aravindhp** [comment]
/hold

Until https://github.com/openshift/windows-machine-config-operator/pull/1637 merges which will unblock the vsphere-e2e-operator test

**Pull request merged**
